test/integration/sandbox-libraries/pm.test.js

```diff
@@ -578,4 +578,322 @@ describe('sandbox library - pm api', function () {
             });
         });
     });
+
+    describe('pm.info properties', function () {
+        describe('pm.info.requestPath', function () {
+            it('should have path as empty array for un-saved requests', function (done) {
```

Comment:
What does "un-saved" mean for `postman-runtime`? It's a Postman App concept leaking here.